lib/AnyEvent/Gearman/Connection.pm

```diff
@@ -68,7 +68,7 @@ sub BUILD {
     my $self = shift;
 
     # parse hostspec
-    my ($host, $service) = parse_hostport $self->hostspec;
+    my ($host, $service) = parse_hostport $self->hostspec, 4730;
     unless (defined $host) {
```

Comment:
What puzzles me is that the `unless` block below is unnecessary anymore. We should assume that `AnyEvent::Socket::parse_hostport` does a better job than merely treating `hostspec` as a hostname if parsing failed.

Reply:
ok.